pages/review-barrier.tsx

```diff
@@ -0,0 +1,365 @@
+                          data-h2-bg-color="b(lightgray)"
+                        >
+                          {intl.formatMessage({
+                            defaultMessage:
+                              "This solution is awaiting confirmation from your manager.",
+                          })}
+                        </p>
+                      </div>
+                      <div style={{ flex: 1 }} data-h2-margin="b(left, l)">
+                        <p>
+                          {" "}
+                          <Link href={""}>
+                            {intl.formatMessage({
+                              defaultMessage: "Review",
+                            })}
+                          </Link>
+                        </p>
+                      </div>
+                    </div>
+                    <p data-h2-margin="b(left, s)">
+                      {intl.formatMessage({
+                        defaultMessage:
+                          "This solution applies to me in following situations:",
+                      })}
+                    </p>
+
+                    <p data-h2-margin="b(left, s)">
+                      {intl.formatMessage({
+                        defaultMessage: "Noisy Workplace",
+                      })}
+                    </p>
+                    <p
+                      data-h2-margin="b(left, none)"
+                      data-h2-padding="b(all, s)"
+                    >
+                      {intl.formatMessage({
+                        defaultMessage:
+                          " Noise cancelling headphones is particularly helpful for me because",
+                      })}
+                      <p>
+                        <PaperClipIcon style={{ width: "1.2rem" }}>
+                          {" "}
+                        </PaperClipIcon>
+                        <Link data-h2-margin="b(left, s)" href={""}>
+                          {intl.formatMessage({
+                            defaultMessage: "my_hearing_assessment.pdf(3MB)",
+                          })}
+                        </Link>
+                      </p>
+                    </p>
+                  </section>{" "}
+                  <h4
+                    data-h2-font-size="b(h3)"
+                    data-h2-margin="b(bottom, none)"
+                  >
+                    {intl.formatMessage(
+                      {
+                        defaultMessage: "Current Solutions",
+                      },
+                      { bold },
+                    )}
+                  </h4>
+                  <p
+                    data-h2-padding="b(all, s)"
+                    data-h2-margin="b(bottom, none)"
+                    data-h2-bg-color="b(lightgray)"
+                  >
+                    {intl.formatMessage({
+                      defaultMessage:
+                        "You haven't marked any solutions as working for this particular barrier.",
```

Comment:
It looks like they want to change this text to: "Physical workplace"
![image](https://user-images.githubusercontent.com/22059495/165823238-9248110d-3f25-47cc-a94c-03172ce32811.png)

Reply:
I don't think so .. this is about example situation in the wireframe 
![Screen Shot 2022-04-28 at 3 16 27 PM](https://user-images.githubusercontent.com/89482910/165829230-bdcbb1dc-70e8-4786-8902-02c7ea2571b2.png)
![Screen Shot 2022-04-28 at 3 17 25 PM](https://user-images.githubusercontent.com/89482910/165829522-655c4c9f-7d0a-49b8-8bd0-89c62f6a466d.png)